### ISSUE TITLE
9/9 Update models in SIM UI

### ISSUE THREAD

**Issue by jamesmrollins** [lzim/teampsd#639]
**1. Add description:** 
Test the update the SP, CC and MM models to handle "corner cases" in the SIM UI.  Promote to PROD if it works ok.

<< Paste screenshots here>>
 
**2. Click on the right:**
a. **Projects** - assign to **issue_tracker.**
b. **Assignee** - assign relevant team members.
c. **Labels**- review and select all potential _MTL_ dependencies.
d. **Milestones** - select the dependent milestone.

-----------------------------------------------------------------------------------------
**Workgroups leads**
**Identify Constraints (check all that apply):**
- [ ]   Capacity
- [ ]   Cost
- [ ]   Schedule
- [ ]   Interference
- [ ]   Capability
- [ ]   Other (describe)

_Add estimated person-hours to complete:_

**Edit the due date (if necessary)**

**Comment by jamesmrollins** [comment]
@saveth I am finished with the update of the models in the TEST session of the SIM UI.  Please let me know when you are done with your functions, and then I can finish the process of updating the model code in GitHub.

@TomRust when I get clearance from @saveth I will send you an updated copy of the CC model.  The variable names for the learning mode settings were incorrect in one of them.  The updated copy should fix that issue.

**Comment by saveth** [comment]
@jamesmrollins what should I be looking out for in the export file?

**Comment by jamesmrollins** [comment]
@saveth .  Since there were no model variable changes, only formula changes, there should be no changes to the export.  So you can go on that, or login and run a module to the end and export.  Then see that it runs OK in your code.  If any exceptions, then we can correct.  I would recommend that latter.

Thanks
James

**Comment by saveth** [comment]
@jamesmrollins I can click on the export in the sim, but in the test environment, the export button doesn't work. I've tried the export from the the expanded "Output and Text" and it does nothing. This is for mm.

**Comment by jamesmrollins** [comment]
@saveth did you save it first?  BTW, which export button did you use?

**Comment by saveth** [comment]
@jamesmrollins I've reviewed cc, mm, and sp as you've requested. I've updated my code for sp export files. Therefore, we can now check off my dependency on this issue. 😄

**Comment by jamesmrollins** [comment]
@saveth thanks!  

@lzim @staceypark @jessfroe  With your permission, I would like to promote to PROD.  I don't believe we have any other dependencies.  Thanks James

**Comment by lzim** [comment]
> @saveth thanks!
> 
> @lzim @staceypark @jessfroe With your permission, I would like to promote to PROD. I don't believe we have any other dependencies. Thanks James

Great, makes sense to promote this.

@TomRust and @staceypark have these been uploaded as the current SP, CC and MM MTL 1.8 models at https://mtl.how?

**Comment by jamesmrollins** [comment]
@saveth @staceypark @lzim @TomRust @jessfroe @ritahitching 

I have updated all files to latest version in all instances of the simulation (DEV/TEST/PROD, and DEMO DEV/TEST/PROD) 

Under the direction of @TomRust , I have cleaned up the model_workgroup directory removing out-of-date files.  I updated the models to the latest and added a model parameters file, so a person would have everything they need to run the models.

I also updated the language in .md files so they reflect the 1.8 models.  Some were still talking about 1.6.
